images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -62,6 +63,71 @@ function Install-Binary
     }
 }
 
+Function Install-VS
```

Comment:
I think that name `Install-VisualStudio` will be clearer.

Reply:
Fixed